ArduPlane/Log.cpp

```diff
@@ -292,7 +298,7 @@ const struct LogStructure Plane::log_structure[] = {
     { LOG_CTUN_MSG, sizeof(log_Control_Tuning),     
       "CTUN", "Qcccchhhf",    "TimeUS,NavRoll,Roll,NavPitch,Pitch,ThrOut,RdrOut,ThrDem,Aspd", "sdddd---n", "FBBBB---0" },
     { LOG_NTUN_MSG, sizeof(log_Nav_Tuning),         
-      "NTUN", "Qfcccfff",  "TimeUS,WpDist,TargBrg,NavBrg,AltErr,XT,XTi,ArspdErr", "smddmmmn", "F0BBB0B0" },
+      "NTUN", "QfcccfffLLi",  "TimeUS,WpDist,TargBrg,NavBrg,AltErr,XT,XTi,ArspdErr,Lat,Lng,Alt", "smddmmmnDUm", "F0BBB0B0GGB" },
```

Comment:
can we fit TLat, TLon and TAlt ?

Reply:
Not unless we cut something else shorter. It's at 63 chars right now.

Reply:
TargBrg -> TBrg?